config/owasp/suppressions.xml

```diff
@@ -14,9 +14,14 @@
     <notes>Temporary Suppression
       CVE-2022-1471
       CVE-2023-28708
+      CVE-2023-20861
+      CVE-2023-20860
```

Comment:
Put these in numerical order (CVE-2023-20860, CVE-2023-20861 then CVE-2023-28708)